clusterloader/README.md

```diff
@@ -0,0 +1,76 @@
+# Cluster Loader
+
+## Building
+In order to vendor in e2e/framework & deps we have used [glide](https://github.com/Masterminds/glide).
+After installing glide just run:
+```
+cd perf-tests/clusterloader/
+glide install -v
+go test -c -o e2e.test
+```
+
+## Running
+Cluster Loader uses e2e/framework as a library so the execution may be familiar. This command is used to test against a local cluster:
+```
+./e2e.test --host="127.0.0.1:8080" --ginkgo.v=true --ginkgo.focus="Cluster Loader" --kubeconfig="$HOME/.kubernetes_auth" --repo-root="$CLPATH" --viper-config=config/test
```

Comment:
seems like we would want a script wrapped/ go wrapped runner vs. raw e2e interface.  

Also repo-root?